project_event/views/project_task_view.xml

```diff
@@ -85,6 +85,20 @@
                         </page>
                         <page string="Technical Sheet">
                             <field name="spectators"/>
+                            <group>
+                                <page string="Time Spent">
+                                    <field name="child_ids">
```

Comment:
we dont need a page if we already have a page named technical sheet. Display the name of the section Time spent.